tools/cold-backup/README.md

```diff
@@ -31,25 +29,22 @@ kubectl create -f ./examples/kafka/kafka-persistent.yaml
 kubectl create -f ./examples/topic/kafka-topic.yaml
 
 # send 100000 messages and consume them
-kubectl run kafka-producer-perf-test -it --image="$CLIENT_IMAGE" \
-  --rm="true" --restart="Never" -- bin/kafka-producer-perf-test.sh \
-  --topic my-topic --record-size 1000 --num-records 100000 --throughput -1 \
-  --producer-props acks=1 bootstrap.servers="my-cluster-kafka-bootstrap:9092"
+kubectl exec -it my-cluster-kafka-0 -c kafka -- \
+  bin/kafka-producer-perf-test.sh --topic my-topic --record-size 1000 --num-records 100000 \
+  --throughput -1 --producer-props acks=1 bootstrap.servers="my-cluster-kafka-bootstrap:9092"
 
 kubectl exec -it my-cluster-kafka-0 -c kafka -- \
   bin/kafka-console-consumer.sh --bootstrap-server :9092 \
   --topic my-topic --group my-group --from-beginning --timeout-ms 15000
 
 # save consumer group offsets
 kubectl exec -it my-cluster-kafka-0 -c kafka -- \
-  bin/kafka-consumer-groups.sh --bootstrap-server :9092 \
-  --group my-group --describe > /tmp/offsets.txt
+  bin/kafka-consumer-groups.sh --bootstrap-server :9092 --group my-group --describe > /tmp/offsets.txt
 
 # send additional 12345 messages
-kubectl run kafka-producer-perf-test -it --image="$CLIENT_IMAGE" \
-  --rm="true" --restart="Never" -- bin/kafka-producer-perf-test.sh \
-  --topic my-topic --record-size 1000 --num-records 12345 --throughput -1 \
-  --producer-props acks=1 bootstrap.servers="my-cluster-kafka-bootstrap:9092"
+kubectl exec -it my-cluster-kafka-0 -c kafka -- \
+  bin/kafka-producer-perf-test.sh --topic my-topic --record-size 1000 --num-records 12345 \
+  --throughput -1 --producer-props acks=1 bootstrap.servers="my-cluster-kafka-bootstrap:9092"
```

Comment:
Why change to `kubectl exec`? That is a bad pattern which should not be encouraged by our docs. It can eaily cause the brokers to get killed etc.

Reply:
Yeah, this is just to avoid updating the image version at every release. I can add a note if you think it is needed.

Reply:
Do we need to update the image every release? I assume it will keep working in general. You can also use a placeholder for the image and let users to fill it in themself.

Reply:
ok